open-api/rest-catalog-open-api.yaml

```diff
@@ -2692,7 +2692,14 @@ components:
           $ref: '#/components/schemas/Schema'
         last-column-id:
           type: integer
-          description: The highest assigned column ID for the table. This is used to ensure columns are always assigned an unused ID when evolving schemas. When omitted, it will be computed on the server side.
+          description:
```

Comment:
I think you can add the `deprecated: true` property for OpenAPI here as well.

Reply:
Nice one, thanks!